### PR TITLE
20220430 preview g2

### PR DESCRIPTION
**Pull request by firstaye-zz**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

**Review by vighatke — APPROVED** [review]

**Comment by firstaye-zz** [comment]
@fengzhou-msft , could you help us to take a look at the PR. I'd like to get it merged soon. Please let us know if there is anything we need to do, thanks!

**Comment by fengzhou-msft** [comment]
@firstaye-zz Track 1 SDK will be deprecated soon and needs approval from @ArthurMa1978 (Arthur Ma). Please send an email to him explaining why you need this update.

**Comment by firstaye-zz** [comment]
cc: @***@***.***>, @***@***.***>
Our release deadline is the end of this month, I'm not sure if we could meet the deadline if we start a new SDK with Track 2 at this point, could we get this one merged and we revisit Track2 later?

 Thanks,
Chris
________________________________

**Comment by firstaye-zz** [comment]
Arthur and Feng,

We (IotHub) are in planning phase of transitioning to Track2, but we haven't started the work yet. So we still need to update Track1 at this time.

Include @Mehmet ***@***.***> @Rahul ***@***.***> for awareness, since we are discussing Track2 a few days ago.


Thanks,
Tao

File: sdk/iothub/Azure.ResourceManager.IotHub/src/autorest.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ csharp: true
 library-name: IotHub
 namespace: Azure.ResourceManager.IotHub
 require: https://github.com/Azure/azure-rest-api-specs/blob/0f9df940977c680c39938c8b8bd5baf893737ed0/specification/iothub/resource-manager/readme.md
-tag: package-2021-07-02
+tag: package-2022-04-30-preview
```
Comment (ArthurMa1978) [inline sdk/iothub/Azure.ResourceManager.IotHub/src/autorest.md:12]: Please remove this, if you want to release a track 2, please establish another pr

File: sdk/iothub/Microsoft.Azure.Management.IotHub/src/Microsoft.Azure.Management.IotHub.csproj [hunk 2]
```diff
@@ -15,7 +15,7 @@
     <AssemblyName>Microsoft.Azure.Management.IotHub</AssemblyName>
     <Version>4.2.0</Version>
     <PackageTags>Microsoft Azure IotHub;IotHub management;IotHub;</PackageTags>
-    <PackageReleaseNotes>Regenerated from an updated 2021-07-02 swagger.</PackageReleaseNotes>
+    <PackageReleaseNotes>Regenerated from an updated 2022-04-30-preview swagger.</PackageReleaseNotes>
```
Comment (ArthurMa1978) [inline sdk/iothub/Microsoft.Azure.Management.IotHub/src/Microsoft.Azure.Management.IotHub.csproj:18]: Please also update the version here from `4.2.0` to `4.3.0-preview`.
Also update the version in  AssemblyInfo.cs file

File: sdk/iothub/Azure.ResourceManager.IotHub/src/autorest.md (hunk 1 above)
Comment (firstaye-zz) [reply sdk/iothub/Azure.ResourceManager.IotHub/src/autorest.md]: reverted

File: sdk/iothub/Microsoft.Azure.Management.IotHub/src/Microsoft.Azure.Management.IotHub.csproj (hunk 2 above)
Comment (firstaye-zz) [reply sdk/iothub/Microsoft.Azure.Management.IotHub/src/Microsoft.Azure.Management.IotHub.csproj:18]: can't postfix with .preview in AssemblyInfo.cs, I made the change to 4.3.0.0

**Review by ArthurMa1978 — APPROVED** [review]

**Pull request merged**